src/lib/components/FileUploader/FileUploader.js

```diff
@@ -106,11 +106,14 @@ export default class FileUploader extends Component {
         </>)}
         <Grid.Row className="file-upload-note-row">
           <Grid.Column width={16}>
-            <Segment basic className="file-upload-note">
-              Note: File addition, removal or modification are not allowed after
-              you have published your upload. This is because a Digital Object
-              Identifier (DOI) is registered with DataCite for each upload.
-            </Segment>
+            <Message visible warning >
```

Comment:
`file-upload-note` class resides in https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme.less#L360. As you aren't using it anymore, we should delete the css rule! Otherwise, it would be hard to know what is still used!

Reply:
To enable merging, I will remove the css class in one of my other PRs in invenio-app-rdm!